projects/js-packages/charts/src/components/pie-chart/__tests__/pie-chart.test.tsx

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
I'm wondering if we could rename the folder to `test` as most of other packages are doing that just for consistency?